metrics/grafana/F3Dashboard.json

```diff
@@ -0,0 +1,2229 @@
+{
```

Comment:
If this is for F3 only, and if this is the export of the current F3 public dashboard then I think we can trim this down significantly.

Also please make sure that the exported JSON is importable into grafana.